### PR TITLE
Add Qibo conversions to documentation

### PR DESCRIPTION
**Pull request by nathanshammah**
Add Qibo conversions to documentation in API-Doc and to Readme.

### CODE REVIEW

**Comment by nathanshammah** [comment]
Looks like Qibo is cut away in the RTD rendering of the Readme page. Trying to shrink all the logo sizes in the table to see if it works.

File: README.md [hunk 1]
```diff
@@ -96,9 +96,9 @@ We refer to any programming language you can write quantum circuits in as a _fro
...
+| <a href="https://quantumai.google/cirq"><img src="https://raw.githubusercontent.com/quantumlib/Cirq/master/docs/images/Cirq_logo_color.png" alt="Cirq logo" width="130"/></a> | <a href="https://qiskit.org/"><img src="https://qiskit.org/images/qiskit-logo.png" alt="Qiskit logo" width="80"/></a> | <a href="https://github.com/rigetti/pyquil"><img src="https://www.rigetti.com/uploads/Logos/logo-rigetti-gray.jpg" alt="Rigetti logo" width="150"/></a> | <a href="https://github.com/aws/amazon-braket-sdk-python"><img src="https://a0.awsstatic.com/libra-css/images/logos/aws_logo_smile_1200x630.png" alt="AWS logo" width="150"/></a> | &nbsp;&nbsp; <a href="https://pennylane.ai/"><img src="https://pennylane.ai/img/xanadu_x.png" alt="PennyLane logo" width="60"/></a> | &nbsp;&nbsp; <a href="https://qibo.science/"><img src="https://raw.githubusercontent.com/qiboteam/qibo/master/doc/source/_static/qibo_logo_dark.svg" alt="Qibo logo" width="120"/></a> |
```
Comment (nathanshammah) [inline README.md:101]:
```suggestion
| [Cirq](https://quantumai.google/cirq)                                                                                                                                         | [Qiskit](https://qiskit.org/)                                                                                         | [pyQuil](https://github.com/rigetti/pyquil)                                                                                                             | [Braket](https://github.com/aws/amazon-braket-sdk-python)                                                                                                                         | [PennyLane](https://pennylane.ai/)                                                                                                  |[Qibo](https://qibo.science/)                                                                                                  |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
| <a href="https://quantumai.google/cirq"><img src="https://raw.githubusercontent.com/quantumlib/Cirq/master/docs/images/Cirq_logo_color.png" alt="Cirq logo" width="65"/></a> | <a href="https://qiskit.org/"><img src="https://qiskit.org/images/qiskit-logo.png" alt="Qiskit logo" width="40"/></a> | <a href="https://github.com/rigetti/pyquil"><img src="https://www.rigetti.com/uploads/Logos/logo-rigetti-gray.jpg" alt="Rigetti logo" width="75"/></a> | <a href="https://github.com/aws/amazon-braket-sdk-python"><img src="https://a0.awsstatic.com/libra-css/images/logos/aws_logo_smile_1200x630.png" alt="AWS logo" width="75"/></a> | &nbsp;&nbsp; <a href="https://pennylane.ai/"><img src="https://pennylane.ai/img/xanadu_x.png" alt="PennyLane logo"/ width="30"/></a> | &nbsp;&nbsp; <a href="https://qibo.science/"><img src="https://raw.githubusercontent.com/qiboteam/qibo/master/doc/source/_static/qibo_logo_dark.svg" alt="Qibo logo" width="60"/></a> |
```

File: README.md [hunk 2]
```diff
@@ -96,9 +96,9 @@ We refer to any programming language you can write quantum circuits in as a _fro
...
+| <a href="https://quantumai.google/cirq"><img src="https://raw.githubusercontent.com/quantumlib/Cirq/master/docs/images/Cirq_logo_color.png" alt="Cirq logo" width="65"/></a> | <a href="https://qiskit.org/"><img src="https://qiskit.org/images/qiskit-logo.png" alt="Qiskit logo" width="40"/></a> | <a href="https://github.com/rigetti/pyquil"><img src="https://www.rigetti.com/uploads/Logos/logo-rigetti-gray.jpg" alt="Rigetti logo" width="75"/></a> | <a href="https://github.com/aws/amazon-braket-sdk-python"><img src="https://a0.awsstatic.com/libra-css/images/logos/aws_logo_smile_1200x630.png" alt="AWS logo" width="75"/></a> | &nbsp;&nbsp; <a href="https://pennylane.ai/"><img src="https://pennylane.ai/img/xanadu_x.png" alt="PennyLane logo"/ width="30"/></a> | &nbsp;&nbsp; <a href="https://qibo.science/"><img src="https://raw.githubusercontent.com/qiboteam/qibo/master/doc/source/_static/qibo_logo_dark.svg" alt="Qibo logo" width="60"/></a> |
```
Comment (nathanshammah) [inline README.md:101]:
```suggestion
| <a href="https://quantumai.google/cirq"><img src="https://raw.githubusercontent.com/quantumlib/Cirq/master/docs/images/Cirq_logo_color.png" alt="Cirq logo" width="65"/></a> | <a href="https://qiskit.org/"><img src="https://qiskit.org/images/qiskit-logo.png" alt="Qiskit logo" width="40"/></a> | <a href="https://github.com/rigetti/pyquil"><img src="https://www.rigetti.com/uploads/Logos/logo-rigetti-gray.jpg" alt="Rigetti logo" width="75"/></a> | <a href="https://github.com/aws/amazon-braket-sdk-python"><img src="https://a0.awsstatic.com/libra-css/images/logos/aws_logo_smile_1200x630.png" alt="AWS logo" width="75"/></a> | &nbsp;&nbsp; <a href="https://pennylane.ai/"><img src="https://pennylane.ai/img/xanadu_x.png" alt="PennyLane logo"  width="30"/></a> | &nbsp;&nbsp; <a href="https://qibo.science/"><img src="https://raw.githubusercontent.com/qiboteam/qibo/master/doc/source/_static/qibo_logo_dark.svg" alt="Qibo logo" width="60"/></a> |
```

**Review by Misty-W — APPROVED** [review]
Nice resizing of the logos, LGTM! 

Docs build failure is unrelated (broken links). The "broken" links still work in Chrome, so I'm re-running the docs build to see if it was due to a temporary outage.

File: docs/source/apidoc.md [hunk 3]
```diff
@@ -365,6 +365,12 @@ See Ref. {cite}`Czarnik_2021_Quantum` for more details on these methods.
 .. automodule:: mitiq.interface.mitiq_pyquil.conversions
    :members:
 ```
+#### Qibo Conversions
+
+```{eval-rst}
+.. automodule:: mitiq.interface.mitiq_qibo.conversions
+   :members:
+```
```
Comment (natestemen) [inline docs/source/apidoc.md:373]: Question: There are a bunch of functions in this module that _maybe_ don't need to go into the API-doc. E.g. all the `..._decomp` functions. Should we just have the `to_qibo` and `from_qibo` functions?

File: docs/source/apidoc.md [hunk 4]
```diff
@@ -365,6 +365,12 @@ See Ref. {cite}`Czarnik_2021_Quantum` for more details on these methods.
 .. automodule:: mitiq.interface.mitiq_pyquil.conversions
    :members:
 ```
+#### Qibo Conversions
+
+```{eval-rst}
+.. automodule:: mitiq.interface.mitiq_qibo.conversions
+   :members:
```
Comment (nathanshammah) [inline docs/source/apidoc.md:372]:
```suggestion
.. autofunction:: mitiq.interface.mitiq_qibo.conversions.from_qibo
.. autofunction:: mitiq.interface.mitiq_qibo.conversions.to_qibo
```

**Comment by nathanshammah** [comment]
Thanks @natestemen, I've addressed the issue, it looks better now ([preview](https://mitiq--2214.org.readthedocs.build/en/2214/apidoc.html#qibo-conversions)).

**Comment by natestemen** [comment]
BTW @nathanshammah please try to use branch name that's descriptive of what the branch contains. A few ideas about how one can do this are available [here](https://hackernoon.com/git-branch-naming-convention-7-best-practices-to-follow-1c2l33g2).

**Pull request merged**